client/locations-list-map.js

```diff
@@ -670,13 +670,17 @@ function getMarkers(data, appliedFilters, bounds, markerOptions) {
           }
         });
         if (hasEntryFilter) {
+          const today = new Date();
+          const dateConversion = 86400000;
```

Comment:
hmm... manual date manipulations are always a little tricky.  For example, how does this interact with leap-years or daylight savings? I tend to think of "dates" as two concepts: calendar dates for human consumptions, and timestamps for calculating deltas between intervals.   In the former, you never do it by hand with math if you can avoid it. In the latter, you ensure everything is in utc or something and as best as possible, convert to seconds (like you did with getTime() to calculate...but even then, there are funny things with leap seconds where dates are no longer monotonic...

Anyways, that's a long way of saying, I believe we re trying to work in calendar dates here, not timestamps.

Is it possible to use inequality operators with the appropriate closed/open intervals?  (note, Javascript date API is broken on equality...cause javascript sucks..:  https://stackoverflow.com/questions/492994/compare-two-dates-with-javascript)

And then use the calendar date functions like https://stackoverflow.com/questions/1296358/subtract-days-from-a-date-in-javascript to generate your interval bounds?